pkg/webui/console/views/application-add/application-add.js

```diff
@@ -140,8 +145,29 @@ const ApplicationAdd = props => {
 }
 
 ApplicationAdd.propTypes = {
+  asConfig: PropTypes.shape({
+    base_url: PropTypes.string,
+  }),
+  jsConfig: PropTypes.shape({
+    base_url: PropTypes.string,
+  }),
   navigateToApplication: PropTypes.func.isRequired,
+  nsConfig: PropTypes.shape({
+    base_url: PropTypes.string,
+  }),
   userId: PropTypes.string.isRequired,
 }
 
+ApplicationAdd.defaultProps = {
+  asConfig: {
+    base_url: undefined,
+  },
+  nsConfig: {
+    base_url: undefined,
+  },
+  jsConfig: {
+    base_url: undefined,
+  },
+}
```

Comment:
If you pass down props as part of the connector, you should rather mark those props as required since you would then not expect these props to be empty. 

But just use the selectors in the component directly (see my other comment).